Mesh_3/include/CGAL/Mesh_3/Mesh_global_optimizer.h

```diff
@@ -200,7 +200,7 @@ class Mesh_global_optimizer_base<Tr, Parallel_tag>
 public:
 
 protected:
-  mutable std::atomic<unsigned int> nb_frozen_points_ = 0;
```

Comment:
A note for a next time, and for education. The original code was:

```c++
std::atomic<unsigned int> nb_frozen_points_ = 0;
```
That code would be accepted in C++17, with the copy-elision mandated by C++17. In C++17, that code is equivalent to:
```c++
std::atomic<unsigned int> temporary_var{0}; // direct initialization of a temporary variable
std::atomic<unsigned int> nb_frozen_points_{temporary_var}; // direct initialization using the copy-constructor
```
and that does not compile because `std::atomic` has no copy-constructor.

As you can see, you can also initialize it directly, with either parenthesis:
```c++
mutable std::atomic<unsigned int> nb_frozen_points_(0);
```
or braces:
```c++
mutable std::atomic<unsigned int> nb_frozen_points_{0u};
```